include/odp/api/spec/packet.h

```diff
@@ -167,6 +167,23 @@ extern "C" {
  * Layer 4 protocol type is ROHC
  */
 
+/**
+ * @typedef odp_packet_chksum_status_t
+ * Checksum check status in packet
+ */
+
+/**
+ * @def ODP_PACKET_CHKSUM_UNKNOWN
+ * Checksum was not checked. Checksum check was not
+ * attempted or the attempt failed.
+ *
+ * @def ODP_PACKET_CHKSUM_BAD
+ * Checksum was checked and it was not correct.
+ *
+ * @def ODP_PACKET_CHKSUM_OK
+ * Checksum was checked and it was correct.
+ */
+
```

Comment:
If you use a syntax like this:
```
/**
 * @enum odp_packet_chksum_status_t
 * Checksum check status in packet
 *
 * @var ODP_PACKET_CHKSUM_UNKNOWN
 * Checksum was not checked. Checksum check was not attempted or the attempt failed.
 *
 * @var ODP_PACKET_CHKSUM_BAD
 * Checksum was checked and it was not correct
 *
 * @var ODP_PACKET_CHKSUM_OK
 * Checksum was checked and it was correct
 */
```
Doxygen seems to document the enum correctly.

Reply:
While you're at it, you could also fix the `odp_packet_color_t` documentation in a similar fashion (in a separate commit).

Reply:
Okay. Will push another commit on top of this for fixing odp_packet_color_t doxygen syntax.